### PR TITLE
Update showcase-videos.tsx

### PR DESCRIPTION
**Pull request by wrf2022**
Added an example (one of many videos created using remotion)

### CODE REVIEW

**Review by MehmetAdemi — APPROVED** [review]
Deleted the second website link since it pointed twice to the same website.

File: packages/docs/src/data/showcase-videos.tsx [hunk 1]
```diff
@@ -1447,6 +1447,25 @@ export const showcaseVideos: ShowcaseVideo[] = [
       name: "Delba Oliveira",
     },
   },
+  {
+    title: "Meet New Books - one of many book recommendation videos",
+    type: "mux_video",
+    muxId: "601PVDW5t02VqRWd4XXDUUBM41t66JVBEO5f00VrGCUXE00",
+    description: "Add a description here",
```
Comment (JonnyBurger) [inline packages/docs/src/data/showcase-videos.tsx]: Can you add a description as well?

**Comment by MehmetAdemi** [comment]
@wrf2022 Thanks for the video and contribution. Very much appreciated! 
FYI: Just recently, we thought about reducing the number of videos on the showcase page since there are many, and it could feel a little overwhelming. We might delete some in the near future. I hope you don't mind if one of them could be yours. 🫢

**Comment by wrf2022** [comment]
Thank you- Really enjoyed working with remotion.
Cheers
Ron

**Pull request merged**